card.js

```diff
@@ -0,0 +1,118 @@
+function Card(title, expierence, image) {
```

Comment:
**Kindly work with the requested feature in your project requirements:**
- In order to associate each project with the popup window details, you must use a JavaScript array to store all of the information for all projects.

The JavaScript code implements an array of objects to store JavaScript objects containing the information for each project.

Check this example:
```
const projects = [
 {
   name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
 }
 {
   name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
 }
]
```

**Reminder:**
- Modal popup data is populated with the data from the projects array.